.github/workflows/ci.yaml

```diff
@@ -7,10 +7,13 @@
 name: CI
 
 # This workflow will run when developer push a topic branch to their
-# fork in github, minimizing noise for maintainers.
+# fork in github, minimizing noise for maintainers;
+# It also runs on nightly basis at 12:00 AM
 on:  # yamllint disable-line rule:truthy
   - push
   - pull_request
+  - schedule:
+      - cron:  '0 0 * * *'
```

Comment:
I see this error in your fork:
https://github.com/Shwetha-Acharya/ramen/actions/runs/5519447447

Try to make this rule work on your fork, maybe try to build once per 15 minutes to be sure it works before changing to dayly build.